com.ibm.streamsx.topology/opt/python/include/splpy.h

```diff
@@ -228,6 +243,26 @@ namespace streamsx {
     }
 
     /*
+    * Call a function passing a Python object
+    * and return the function return as a boolean
+    * nb. the GILLock must be held
+    */
+    static int pyTupleFilter(PyObject * function, PyObject * arg) {
```

Comment:
The existing pyTupleFilter can be modified to just call this function, rather than having duplicated code. Much like the older _pyTupleFilter.